test/e2e/support/reports.js

```diff
@@ -1,15 +1,15 @@
-const reporter = require('cucumberjs-allure-reporter');
-const fs = require('fs');
-const mkdirp = require('mkdirp');
+// const reporter = require('cucumberjs-allure-reporter');
+// const fs = require('fs');
+// const mkdirp = require('mkdirp');
 
-const xmlReports = `${process.cwd()}/reports/xml`;
+// const xmlReports = `${process.cwd()}/reports/xml`;
 
-// var createXmlDir = (function() {
-//     if (!fs.existsSync(xmlReports)) {
-//         mkdirp.sync(xmlReports);
-//     }
-// })();
+// // var createXmlDir = (function() {
```

Comment:
Same for these commented-out lines, too.

Reply:
yup however we are just validating if the pr actually pass as cucumber was now causing the new errors so temporary

Reply:
OK, no worries - will mark all these as resolved.